changelog/5636.bugfix.rst

```diff
@@ -0,0 +1,4 @@
+Update dependencies based on the ``dependabot`` check.
```

Comment:
should this be an `improvement` changelog?

Reply:
👍